node.gyp

```diff
@@ -322,6 +322,7 @@
           'dependencies': [
             'deps/v8_inspector/third_party/v8_inspector/platform/'
                 'v8_inspector/v8_inspector.gyp:v8_inspector_stl',
+            'v8_inspector_compress_protocol_json#host',
```

Comment:
The previous line is a continuation of the one before it.  Note how the first line doesn't have a comma at the end.